session.md

```diff
@@ -294,6 +294,9 @@ If none of the existing session drivers fit your application's needs, Laravel ma
 > [!NOTE]  
 > Laravel does not ship with a directory to contain your extensions. You are free to place them anywhere you like. In this example, we have created an `Extensions` directory to house the `MongoSessionHandler`.
 
+> [!NOTE]
+> We use MongoDB as an example, but if you are looking for a MongoDB session driver, you can use the `cache` session driver with a `mongodb` cache store. Read more about the [MongoDB Cache Driver](https://www.mongodb.com/docs/drivers/php/laravel-mongodb/current/cache/).
+
```

Comment:
I didn't realize this earlier, but did the Laravel docs already include a MongoSessionHandler example before any of our contributions. Neat if so.

I assume we don't need to mention the `mongodb/laravel` package here as you did so earlier on the page.